### DISCUSSION TITLE
How does vue3 use @wagmi/core to configure the connection to the okx wallet?

### DISCUSSION THREAD

**Discussion by percyzt** [wevm/wagmi#2704] [Q&A]
How does vue3 use @wagmi/core to configure the connection to the okx wallet?

**Comment by unicape** [comment]
@youthinn Here is the vue version https://github.com/unicape/use-wagmi

**Reply by KianYang-Lee** [reply]
This is `vue 2.X` I believe he's asking for `vue 3.X`
